### ISSUE TITLE
Insufficient labels on some links for screen reader accessibility

### ISSUE THREAD

**Issue by tech10** [linode/manager#6450]
When using a screen reader on version 1.9.0, some dashboard links are labeled in a manner that makes accessing a specific linode, domain, or other item on the dashboard difficult. Browser and operating system used doesn't seem relevant for this issue.

Two examples. Under the sections on the dashboard for linodes and domains, each link for a linode or a domain simply possesses the alt text label of, View Details. If all such links for linodes, domains, volumes, NodeBalancers, and anything that might be added to the dashboard in the future could be labeled with the label of such items, or what domain is being linked to, this would add more accessibility for screen reader access on the dashboard.

A note that, when going to the individual sections to linodes or domains, and presumably other items as well, all links to linodes and domains are more accurately labeled for screen reader accessibility.

**Comment by WilkinsKa1** [comment]
Hi @tech10, thanks for reaching out to us with this feedback. I just put up a [PR](https://github.com/linode/manager/pull/6452) to address this issue.

**Comment by tech10** [comment]
Excellent. I have found more links labeled as View Details on the users tab of the account page. These could probably be better labeled with the username.

**Comment by WilkinsKa1** [comment]
@tech10 thanks for this info- I just updated my PR to make adjustments across the app for this issue, including the sections you recently mentioned.
